docs/contributing.md

```diff
@@ -0,0 +1,52 @@
+## Contributing to Toltec
+
+Thank you for taking time to contribute to Toltec!
+We welcome contributions from anyone, regarding for example [reporting bugs](#reporting-a-bug), [requesting](#request-a-package) or [adding new packages](#adding-a-new-package), [upgrading existing ones](#upgrading-a-package), [improving the docs](#improving-the-documentation), or other topics.
+
+To make a request or report a bug, you simply need to [open a new issue](../../../issues/new/choose).
+
+To directly propose changes, the basic procedure is to fork this repository, make the desired changes in your newly created local copy, and open a pull request.
+When proposing changes, please make sure that you follow the [style guide](#style-guide).
+After you submit your pull request, a maintainer will take time to review your changes and merge them into the repository if they fit.
```

Comment:
```suggestion
After you submit your pull request, a maintainer will take time to review your changes, request modifications and then merge your changes into the repository if they fit.
```

I feel like "fit" is ending the sentence too early - what are the changes supposed to fit?

Reply:
I was not sure how to express concisely that the reviewer can also choose to reject the changes if they feel that they don’t fit with the project goals.